example/hyperparam/README.rst

```diff
@@ -0,0 +1,60 @@
+Hyper Parameter Tuning Example
```

Comment:
Nit: I think we should say "hyperparameter" (one word) throughout. Right now we have a mix of things: hyper parameter, hyperparameter, hyper param, etc. Just stick with one.

Reply:
Makes sense.